repo/blob/s3/s3_options.go

```diff
@@ -29,4 +30,7 @@ type Options struct {
 
 	// PointInTime specifies a view of the (versioned) store at that time
 	PointInTime *time.Time `json:"pointInTime,omitempty"`
+
+	// Creds is an optional credential container to use
+	Creds *credentials.Credentials `json:"-"`
```

Comment:
Unfortunately doesn't look like this works well because we will at some point serialize this config to a file and read from there.